app/code/Transoft/Blog/Helper/Data.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Remove helper. Use Magento\Store\Model\ScopeInterface::getValue(...) where you need.

Reply:
fixed